src/gl/quadFeature.js

```diff
@@ -423,6 +423,7 @@ capabilities[quadFeature.capabilities.image] = true;
 capabilities[quadFeature.capabilities.imageCrop] = true;
 capabilities[quadFeature.capabilities.imageFixedScale] = false;
 capabilities[quadFeature.capabilities.imageFull] = true;
+capabilities[quadFeature.capabilities.canvas] = false;
```

Comment:
My only concern is that now we need something similar in canvas version? my slight preference would be not to list canvas in gl and gl in canvas (and what about d3).